### PR TITLE
chore: Add README and repository configuration files

### PR DESCRIPTION
**Pull request by validcube**
Align the repository to others within the organisation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,110 @@
...
 
-## Building
+Some of the features the website provide are:
 
-To create a production build, run:
+* ⚙️ **Powered by ReVanced API**: Utilize most of the features provided by [ReVanced API](https://github.com/ReVanced/revanced-api)
+* 📢 **Announcement**: Easy way to create and view announcements to users on the internet
+* ♿ **Accessibility-friendly**: Accessible by anyone who needs an accessibility feature
+* 🔎 **Patch viewer**: List all the patches available in [ReVanced Patches](https://github.com/ReVanced/revanced-patches)
+* ✨ **And much more!**
```
Comment (oSumAtrIX) [inline README.md:81]: Features should be: Download latest version of manager, view patches, explore donation methods, view contributors, about and links of the hoster

File: README.md [hunk 2]
```diff
@@ -1,26 +1,110 @@
...
 
-The official ReVanced Website.
+# 🌐 ReVanced Website
 
-## Developing
+![Deployment status](https://img.shields.io/github/actions/workflow/status/ReVanced/revanced-website/deploy.yml)
+![GPLv3 License](https://img.shields.io/badge/License-GPL%20v3-yellow.svg)
 
-Install dependencies with `npm install` (or `pnpm install`).
+This repository housed the content of ReVanced Website.
```
[the remark this answers is not in the document]
Comment (oSumAtrIX) [reply README.md:66]:
```suggestion
This repository contains the source code of ReVanced Website.
```

File: README.md [hunk 3]
```diff
@@ -1,26 +1,110 @@
...
 
-Start a development server by running:
+## ❓ About
 
-```bash
-npm run dev
+Website is a site built from Svelte that uses ReVanced API to quickly provides 
+way to retrieve information regarding announcement, patches, contributor, and much more.
```
Comment (oSumAtrIX) [inline README.md:71]:
```suggestion
ReVanced Website is the front page of ReVanced. It is built using Svelte and uses [ReVanced API](https://github.com/ReVanced/revanced-api) as a data source.
```

File: README.md [hunk 4]
```diff
@@ -1,26 +1,110 @@
...
+* 🔎 **Patch viewer**: List all the patches available in [ReVanced Patches](https://github.com/ReVanced/revanced-patches)
+* ✨ **And much more!**
 
-```bash
-npm run build
-```
+## 🚀 How to get started
 
-You can preview the production build with `npm run preview`.
+To use ReVanced Website, go to https://revanced.app
```
Comment (oSumAtrIX) [inline README.md:85]: This section should show setup & build instructions (.env setup & build command)

Comment (oSumAtrIX) [inline README.md:85]: Maybe

```suggestion
ReVanced Website can be deployed locally using `npm run dev`. 
```

File: README.md [hunk 5]
```diff
@@ -1,26 +1,110 @@
...
+
+Thank you for considering contributing to ReVanced Patches. You can find the contribution guidelines [here](CONTRIBUTING.md).
+
+### 🛠️ Building
+
+To build ReVanced Website, [Node.js](https://nodejs.org/en/download) and Git must be installed.  
+Follow the steps below to build ReVanced Website:
+
+1. Run `git clone git@github.com:ReVanced/revanced-website.git` to clone the repository
+2. Run `npm install` or `pnpm install` to install the dependencies
```
Comment (oSumAtrIX) [inline README.md:99]: pnpm lock should be removed, its not really being updated.

```suggestion
2. Run `npm i` to install the dependencies
```

File: README.md [hunk 6]
```diff
@@ -1,26 +1,110 @@
...
+2. Run `npm run build` to build the project
```
Comment (oSumAtrIX) [inline README.md:100]: The `.env` has to be setup according to the .env.example

File: README.md [hunk 7]
```diff
@@ -1,26 +1,110 @@
...
+
+> ![TIP]
+> To start development server run `npm run dev` or `npm run preview` to preview production build
```
Comment (oSumAtrIX) [inline README.md:103]: This tip can be removed/moved to get started maybe.

File: README.md [hunk 8]
```diff
@@ -1,26 +1,110 @@
...
+
+## 📜 Licence
```
[the remark this answers is not in the document]
Comment (oSumAtrIX) [reply README.md]: We may need to apply this change in all repos though, but not in this PR to keep it consistent.

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]
Rebase this branch on dev

File: README.md (hunk 1 above)
Comment (validcube) [reply README.md]: Done, only 4 of the features are listed for consistency sake.

File: README.md (hunk 4 above)
Comment (validcube) [reply README.md]: Applied, the change also moved the tip block to this section.

File: README.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (validcube) [reply README.md:103]: > We may need to apply this change in all repos though, but not in this PR to keep it consistent.

I can open PR across the ReVanced repository for this, would you like me to do it?

[the remark this answers is not in the document]
Comment (oSumAtrIX) [reply README.md:103]: It's easier if I directly commit to dev in all the repos later

**Comment by validcube** [comment]
The change seems to be finished, I'm not sure what is left to be done now.

**Pull request merged**